client/app/scripts/reducers/root.js

```diff
@@ -721,6 +724,34 @@ export function rootReducer(state = initialState, action) {
       return state.set('showingTroubleshootingMenu', !state.get('showingTroubleshootingMenu'));
     }
 
+    case ActionTypes.TOGGLE_CONTRAST_MODE: {
+      const modules = [
+        require.resolve('../../styles/main.scss'),
+        require.resolve('../../styles/contrast.scss')
+      ];
+      // Bust the webpack require cache to for a re-download of the stylesheets
+      modules.forEach((i) => {
+        const children = require.cache[i] ? require.cache[i].children : [];
+        children.forEach((c) => {
+          delete require.cache[c];
+        });
+        delete require.cache[i];
+      });
+
+      const contrastEnabled = state.get('contrastMode');
+      if (!contrastEnabled || action.enabled) {
```

Comment:
> "If contrast mode is currently or if the user explicitly says they want to turn it on, turn it on"

My understanding is that this sentence translates to `contrastEnabled || action.enabled`, in which case there's an extra `!` at the beginning of your *if*-clause, but even then I'm not sure why we should care for `contrastEnabled` part in here at all.

The current conditions seem to overlap whenever `contrastEnabled === action.enabled` in which case the contrast styles would always take advantage, which seems a bit arbitrary. The code seems to work just fine though and I'm not sure why :D But I still believe these conditions could be simplified a bit! :)